### DISCUSSION TITLE
What does the EIN represent

### DISCUSSION THREAD

**Discussion by aaronmefford** [CMSgov/price-transparency-guide#620] [General]
When reviewing these files and looking at the NPIs grouped under a specific EIN there are cases where there appears to be no association between the NPIs that appear in the group.

Our assumption had been that the EIN would represent at single entity that contracted at a specific rate, and the NPIs associated with that contract.  Is this an accurate assumption or what should I understand these groups to represent? What is the value under TIN expected to represent?

**Comment by hw-monkey** [comment]
EIN = Employer Identification Number

Specification state:
____________________________________________
For most businesses reporting cases, a tax identification number (tin) is used to represent a business. There are situations where a provider's social security number is still used as a tin. In order to keep private personally identifiable information out of these files, substitute the provider's npi number for the social security number. When a npi number is used, it is assumed that the provider would otherwise be reporting by their social security number.

In contractual arrangements that are only made at the TIN level, where NPIs are unknown or otherwise unavailable, the value "0" should be reported for the NPI field. In contractual arrangements where both the NPI and TIN are available, both are required to be disclosed.

Clarification stated:
__________________________________________________
**In contractual arrangements where both the NPI and TIN are available**, both are required to be disclosed.

I suspect the legal interpretation of the BOLD is where you will find things differ from expectations.

**Reply by aaronmefford** [reply]
I perhaps didnt ask the question as well as I would have liked.  Your answer seems to indicate my expectation, that the EIN should be the tax id for the entity which contracted with the payer network for that specific negotiated rate, and the NPIs where available are the NPIs that operate under that contractual agreement.

The observation however has been that while many times this appears to be the case at other times the NPIs point to many different organizations who do not appear to have any relationship.   This is what I am trying to understand.   If it is the tax id of the contracting organization then the npis should be operating under the same organization.

**Reply by AshleyOboe** [reply]
I have the same question... but my hypothesis is that this is pulled from claims data and that the NPIs may be attending, referring, etc and the EIN comes from the billing NPI.  So instead of pulling just from contract data (which we also don't think is as granular as the CMS reporting requirements for most payers) they are using actual allowed amounts (aka in-network negotiated rates) from the claims  in the reporting.  That may be why we are also seeing some outliers on the high and low end, where they may have made some catch-up payments on a claim.
